### ISSUE TITLE
Conduct Interviews and Compile Research for Participation Lab 

### ISSUE THREAD

**Issue by LucyeoH-zz** [mozilla/participation-org#114]
## Goal:

To have solid data for each of the five Participation Lab insights that will set us up to create a report with conclusions. 
## Success:

We have a large amount of high quality data that provides interesting insights into the four topics we're looking into. 
## Related:

Participation Lab Strategy - [link](https://docs.google.com/a/mozilla.com/document/d/1YaADpq7S9pma6oZF3ZxjgXNIU118cogiHsdPrHf1cDM/edit?usp=sharing]
[Reference meta issues and past issues) 
#85
## Roles:

Owner: @lucyeoh
Involved: @emmairwin 
## Required:

Topic 1 - Swag Contribution
- [ ] Interview Reps about swag ( @emmairwin [small, medium, large], Due: 27) 
- [x] Interview Sarah Allen about Maker Party & swag (@lucyeoh [small] Due: Aug 28th -> Sept 2) 
- [x] Interview FSA about swag (@lucyeoh [small] Due: Aug 28th -> Sept 3rd) 

Topic 2 - Contributor vs. Volunteer
- [ ] Talk to Holly about volunteer word findings (@lucyeoh [small], Due: ?) 
- [x] Emma to interview 9 people about volunteer vs. contributor (@emmairwin [medium] Due: Aug 24th)

Topic 3 - Login as a barrier
- [x] Talk to Scott about Webmaker and Fundraising login decisions (@lucyeoh, [small], Due: Aug 24th) 
- [x] Look into One and Done numbers (@lucyeoh, [small], Due: Aug 28th) 

Topic 4 - What Makes People Finish tasks
- [ ] Look at One and Done numbers before verification added (@lucyeoh [small] Due: Sept 4th)
- [ ] Look at number of tasks completed and fake entries after verfication layer is added (@lucyeoh [small] Sept 10th)
- Talk to similar groups i.e. Mike Hoye to see if they have numbers/hypothesis related to task completion (@emmairwin [small] Due: Next Quarter ) 

Topic 5 - Formal Outreach Programs for Volunteer (Nice to have for this quarter) 
- nterview Ateam about their program (@owner, [small, medium, large], Due: Aug 7th) 
- Interview MozCamps about their program (@owner, [small, medium, large], Due: Aug 7th) 
- interview DevRel about their program (@owner, [small, medium, large], Due: Aug 7th) 
- Follow Participation MozFest recruitment process (@owner, [small, medium, large], Due: Aug 7th) 
- Identify other teams around the org who are running similar programs (@owner, [small, medium, large], Due: Aug 7th)

**Comment by LucyeoH** [comment]
![racoon gif](https://cloud.githubusercontent.com/assets/8070833/9473913/627d307e-4b2b-11e5-90a7-af8f8c44c956.gif)

**Comment by LucyeoH-zz** [comment]
Closing Notes: 
- Don't know if Emma was able to talk to reps before she left
- Wanted to wait to resolve some contribute page stuff before discussing volunteer wording
- Got blocked on permission to access their more detailed metrics [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1201678)
- Will prioritize this next heartbeat

**Comment by brianking** [comment]
Blockers
- waiting to talk to Holly about contribute page
- 1 and Done numbers, asked for permissions

Whole topic 5 is nice-to-have.
